pkg/backup/file_backend_test.go

```diff
@@ -22,6 +22,49 @@ import (
 	"testing"
 )
 
+func TestFileBackendGetLatest(t *testing.T) {
+	names := []string{
+		makeBackupName("3.0.1", 12),
```

Comment:
> we ignore version

For this point, can we change `makeBackupName("3.0.0", 1),` to `makeBackupName("3.1.0", 1),` and add a comment "ignore version"?